tree-sitter-darklang/test/corpus/exhaustive/exprs/let_expr.txt

```diff
@@ -0,0 +1,170 @@
+==================
+simple let_expression
+==================
+
+let x = 1L
+x
+
+---
+
+(source_file
+  (expression
+    (let_expression
+      (keyword) (variable_identifier) (symbol)
+      (expression (simple_expression (int64_literal (digits (positive_digits)) (symbol))))
+      (expression (simple_expression (variable_identifier)))
+    )
+  )
+)
+
+
+==================
+simple let_expression -with indent
+==================
+
+let x =
+  1L
+x
+
+---
+
+(source_file
+  (expression
+    (let_expression
+      (keyword) (variable_identifier) (symbol)
+      (indent)
+      (expression (simple_expression (int64_literal (digits (positive_digits)) (symbol))))
+      (dedent)
+      (expression (simple_expression (variable_identifier)))
+    )
+  )
+)
```

Comment:
This looks wrong to me. I'd expect this I think:

```suggestion
==================
simple let_expression -with indent
==================

let x =
  1L
x

---

(source_file
  (expression
    (let_expression
      (keyword) (variable_identifier) (symbol)
      (indent)
      (expression (simple_expression (int64_literal (digits (positive_digits)) (symbol))))
      (dedent)
    )
    (expression (simple_expression (variable_identifier)))
  )
)
```

Reply:
Hmm, why? 
`x` is the body of the `let_expression` so it should be included within it, no?

Reply:
you're right, I was confusing myself.